scipy/stats/_distn_infrastructure.py

```diff
@@ -2339,14 +2339,30 @@ def integ(x):
     def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
                conditional=False, **kwds):
         """Calculate expected value of a function with respect to the
-        distribution.
+        distribution by numerical integration.
 
         The expected value of a function ``f(x)`` with respect to a
         distribution ``dist`` is defined as::
 
-                    ubound
-            E[x] = Integral(f(x) * dist.pdf(x))
-                    lbound
+                    ub
+            E[f(x)] = Integral(f(x) * dist.pdf(x)),
+                    lb
+
+        where ``ub`` and ``lb`` are arguments and ``x`` has the ``dist.pdf(x)`` 
+        distribution. If the bounds ``lb`` and ``ub`` correspond to the 
+        support of the distribution, e.g. ``[-inf, inf]`` in the default 
+        case, then the integral is the unrestricted expectation of ``f(x)``. 
+        Also, the function ``f(x)`` may be defined such that ``f(x)`` is ``0`` 
+        outside a finite interval in which case the expectation is 
+        calculated within the finite range ``[lb, ub]``. 
+
+        To understand the effect of the bounds of integration consider the
```

Comment:
probably better to move this part to notes below or create a section example

Reply:
I understand you mean the "To understand the effect of the bounds..." part to be moved down into the notes section-agreed.